sencha-workspace/SlateDemonstrationsStudent/app/view/Dashboard.js

```diff
@@ -170,16 +170,18 @@ Ext.define('SlateDemonstrationsStudent.view.Dashboard', {
         me.fireEvent('selectedcontentareachange', me, contentArea, oldContentArea);
     },
 
-    updateHasEnrollments: function(enrolled) {
```

Comment:
**question (non-blocking)** Does this all need to re-run every time we call `setHasEnrollments`, even if the passed value hasn't changed?

The answer could be "yes" if any of the subcomponents this toggles the visibility of also have other ways they could get their visibility toggled from—that would necessitate an "enforce this state every time" approach that you get with `apply`. 

If this config is the authoritative boss of the visibility of all these subcomponents though, we can use the `update` method to get "only apply this state when the input changes". In a case where the config handler only relays values to a bunch of sub-component config handlers the difference doesn't matter much and you can err on the side of pushing the state out, but its always a good idea to minimize layout calls

If you had a reason using apply instead improved experience, then disregard this. It might be good practice to leave a comment why apply needs to be used since someone might think to "fix" it later